src/character.cpp

```diff
@@ -739,6 +740,65 @@ void Character::mount_creature( monster &z )
     mod_moves( -100 );
 }
 
+bool Character::check_mount_will_move( const tripoint &dest_loc )
+{
+    if( !is_mounted() ) {
+        return true;
+    }
+    if( mounted_creature && mounted_creature->type->has_fear_trigger( mon_trigger::HOSTILE_CLOSE ) ) {
+        for( const monster &critter : g->all_monsters() ) {
+            Attitude att = critter.attitude_to( *this );
+            if( att == A_HOSTILE && sees( critter ) && rl_dist( pos(), critter.pos() ) <= 5 &&
+                rl_dist( dest_loc, critter.pos() ) < rl_dist( pos(), critter.pos() ) ) {
+                add_msg_if_player( _( "Your %s refuses to move closer to the %s!" ), mounted_creature->get_name(),
```

Comment:
Could change this to "You cannot move as your %s isn't able to move." for the time being.

Reply:
Another option is, "You fail to budge your %s!"

Reply:
The implication to the player that the mount is refusing to go closer due to enemies nearby is more informative, otherwise they might think it wont move due to other reasons, is there a reason youd want it more general and less specific?

Reply:
To not add a new string before 0.E.

Reply:
Yes after 0.E we'd make it more specific, but I *can't* merge this before 0.E if it adds new strings.

Reply:
Ahhh so that is the same string as something that already exists translated.